js/todo.js

```diff
@@ -0,0 +1,110 @@
+const toDoInput = document.querySelector('input.toDoInput');
+const toDoNum = document.querySelector('.toDoNum');
+const doneNum = document.querySelector('.doneNum');
+const submitForm = document.querySelector('form');
+const toDoList = document.querySelector('.toDoList');
+const doneList = document.querySelector('.doneList');
+
+let toDos = [];
+
+//todo 카운팅
+function toDoCount() {
+  const toDoLeft = toDos.filter((toDo) => toDo.isCompleted === false);
+  toDoNum.innerHTML = `(${toDoLeft.length})`;
+}
```

Comment:
es6 문법인 arrow function을 써 보셔도 좋을 것 같아요

```suggestion
const toDoCount = () => {
  const toDoLeft = toDos.filter((toDo) => toDo.isCompleted === false);
  toDoNum.innerHTML = `(${toDoLeft.length})`;
}
```

Reply:
넵!! 화살표 함수와 친해지도록 노력해야겠습니다 !